playbooks/roles/networking/templates/rpc-on-rpc.j2

```diff
@@ -40,7 +40,7 @@ iface br-vxlan inet static
 {% endif %}
 
 {% if storage_ip is defined %}
-{% if inventory_hostname in groups.compute %}
+{% if inventory_hostname in groups.infra or inventory_hostname in groups.compute %}
```

Comment:
It's possible to shorten this to:

```
{% if inventory_hostname in [groups.infra, groups.compute] %}
```

Not a big deal, but I thought it was worth mentioning. ;)